docs/guide/writing-tasks.md

```diff
@@ -371,7 +371,7 @@ Note that `pip` cache won't be uploaded in this example: using `upload_caches` d
 
 An `artifacts` instruction allows to store files and expose them in the UI for downloading later. An `artifacts` instruction
 can be named the same way as `script` instruction and has only one required `path` field which accepts a [glob pattern](https://en.wikipedia.org/wiki/Glob_(programming))
-of files relative to `$CIRRUS_WORKING_DIR` to store. Right now only storing files under [`$CIRRUS_WORKING_DIR` folder](#environment-variables) as artifacts is supported with a total size limit of 1G for a community task and with no limit on your own infrastructure.
+of files relative to `$CIRRUS_WORKING_DIR` to store. Right now only storing files under [`$CIRRUS_WORKING_DIR` folder](#environment-variables) as artifacts is supported with a total size limit of 1G for a free task and with no limit on your own infrastructure.
```

Comment:
```suggestion
of files relative to `$CIRRUS_WORKING_DIR` to store. Right now only storing files under [`$CIRRUS_WORKING_DIR` folder](#environment-variables) as artifacts is supported with a total size limit of 1G on Cirrus Cloud Clusters task and with no limit on your own infrastructure.
```

Reply:
There is 5Gb limit for one using compute credits. 🤔